### PR TITLE
[Test] Digital order taxes based on billing country tax rate

### PR DESCRIPTION
**Pull request by szczecha**
I want to merge this change because it covers test [CORE_2008](https://saleor.testmo.net/repositories/5?group_id=461&case_id=4730) Digital Order calculate simple taxes based on Billing address

# Impact

- [ ] New migrations
- [ ] New/Updated API fields or mutations
- [ ] Deprecated API fields or mutations
- [ ] Removed API types, fields, or mutations

# Docs

- [ ] Link to documentation:

# Pull Request Checklist

- [ ] Privileged queries and mutations are either absent or guarded by proper permission checks
- [ ] Database queries are optimized and the number of queries is constant
- [ ] Database migrations are either absent or optimized for zero downtime
- [ ] The changes are covered by test cases

### CODE REVIEW

File: saleor/tests/e2e/orders/utils/draft_order_create.py [hunk 1]
```diff
@@ -9,41 +9,42 @@
...
-            unitPrice {
-                gross {
-                    amount
-                }
-            }
-            totalPrice {
-                gross {
-                    amount
-                }
-            }
```
Comment (yellowee) [inline saleor/tests/e2e/orders/utils/draft_order_create.py:45]: I am not sure this should be removed - have you checked those fields are unused?

File: saleor/tests/e2e/product/utils/preparing_product.py [hunk 2]
```diff
@@ -59,3 +60,55 @@ def prepare_product(
     ]["amount"]
 
     return product_id, product_variant_id, product_variant_price
+
+
+def prepare_digital_product(
```
Comment (yellowee) [inline saleor/tests/e2e/product/utils/preparing_product.py:65]: Shouldn't it be added to __init__.py?

File: saleor/tests/e2e/orders/utils/draft_order_create.py (hunk 1 above)
Comment (szczecha) [reply saleor/tests/e2e/orders/utils/draft_order_create.py:45]: I only added `isShippingRequired` and updated formatting (copied and pasted form GraphQL playground. If you take a look at the whole [file](https://github.com/saleor/saleor/blob/87ff6b933879e8f9e5b0e28b69fade1911e702a2/saleor/tests/e2e/orders/utils/draft_order_create.py) on this branch all fields are still here.

File: saleor/tests/e2e/product/utils/preparing_product.py (hunk 2 above)
Comment (szczecha) [reply saleor/tests/e2e/product/utils/preparing_product.py:65]: I added `prepare_digital_product` and `prepare_product` to __init__.py but in this PR I will use `from ..product.utils import prepare_digital_product` notation only for this test and update the rest test in separate PR.

**Review by yellowee — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

**Pull request merged**